tasks/cli_tools.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+# Compile available themes
```

Comment:
(Cosmetic) Having moved this section back to tasks/build.yml, should remove this comment line.

Reply:
Good catch!